.gitignore

```diff
@@ -1,7 +1,4 @@
 target/
-.project
-.settings
-.classpath
 .idea
```

Comment:
These are useful for people using Eclipse IDE.

Reply:
Ah, ok.